src/pybricks/_common.pyi

```diff
@@ -80,20 +79,6 @@ class Motor(DCMotor):
     ) -> int: ...
     def track_target(self, target_angle: int) -> None: ...
 
-class Speaker:
```

Comment:
This class should stay here and be modified to reflect what is implemented on SPIKE Prime.